### PR TITLE
feat(organization): migrate billing to react query 

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-844

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> There is no better time than now, better future does not exist

File: libs/pages/events/src/lib/ui/row-event/row-event.tsx [hunk 1]
```diff
@@ -89,6 +89,7 @@ export function RowEvent(props: RowEventProps) {
       [OrganizationEventTargetType.WEBHOOK]: () => customLink(SETTINGS_URL(organizationId) + SETTINGS_WEBHOOKS),
       [OrganizationEventTargetType.CONTAINER_REGISTRY]: () =>
         customLink(SETTINGS_URL(organizationId) + SETTINGS_CONTAINER_REGISTRIES_URL),
+      [OrganizationEventTargetType.HELM]: () => <div />,
```
Comment (RemiBonnet) [inline libs/pages/events/src/lib/ui/row-event/row-event.tsx:92]: Fix needed with the update of api doc

File: .eslintrc.json [hunk 2]
```diff
@@ -132,6 +132,11 @@
         "react/jsx-curly-brace-presence": "error",
         "react/jsx-key": "error"
       }
+    },
+    {
+      "files": ["*.ts", "*.tsx"],
```
Comment (ctjhoa) [inline .eslintrc.json:137]: We already have a `"files": ["*.ts", "*.tsx"]` line 36

File: libs/pages/events/src/lib/ui/row-event/row-event.tsx (hunk 1 above)
Comment (ctjhoa) [reply libs/pages/events/src/lib/ui/row-event/row-event.tsx:92]: return `null` instead

**Review by ctjhoa — CHANGES_REQUESTED** [review]

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**